common/merkle/src/lib.rs

```diff
@@ -47,3 +54,43 @@ fn merge(left: &Hash, right: &Hash) -> Hash {
     root.extend_from_slice(right);
     Hasher::digest(Bytes::from(root))
 }
+
+pub struct TrieMerkle(PatriciaTrie<MemoryDB, HasherKeccak>);
+
+impl Default for TrieMerkle {
+    fn default() -> Self {
+        TrieMerkle(PatriciaTrie::new(
+            Arc::new(MemoryDB::new(false)),
```

Comment:
why it must be arc here?

Reply:
This is required by the `cita_trie` crate.